cloudconfig/containerinit/container_userdata.go

```diff
@@ -380,3 +385,103 @@ else
 fi`[1:],
 		oldInterfacesFile, newInterfacesFile)
 }
+
+func populateNetworkInterfaces(networkFile string) string {
+	s := `
+if [ -f /usr/bin/python ]; then
+    python %s.py --interfaces_file %s
+else
+    python3 %s.py --interfaces_file %s
+fi
+`
+	return fmt.Sprintf(s, networkFile, networkFile, networkFile, networkFile)
+}
+
+func PopulateNetworkInterfacesScript() string {
+	s := `import subprocess, re, argparse, os, time
```

Comment:
So a quick terminal test says
```
>>> print 'foo'
foo
>>> x = print
  File "<stdin>", line 1
    x = print
            ^
SyntaxError: invalid syntax
>>> from __future__ import print_function
>>> x = print
>>>
```
It just happens that:
print ("bar")
is valid in python because (1) is just (1)
If you did:
print ("bar",)
Then without the __future__ you get:
```
>>> print("bar",)
('bar',)
>>> from __future__ import print_function
>>> print("bar",)
bar
```
Python 2: print ("bar",)
means print this tuple which contains bar
Python 3: print("bar",)
means call this function with first argument "bar".

So while all of our print statements are of the form ("foo") which is not a tuple because it doesn't have more than one argument, its more correct to import the print_function.